mne/tests/test_simulation.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 10 12:33:57 2020
+
+@author: Jeff Stout
+
+This code only tests for a specific bug found in the code.  Further test
+functions should be written to test the full simulation code.
+"""
+
+
+import os.path as op
+import copy
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+
+def test_simulation_cascade():
```

Comment:
Rather than creating a new file, could you put this in `mne/simulation/tests/test_raw.py`?

Reply:
Added test into the specified file and git rm-ed the mne/tests/test_simulation.py